glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/m3db/m3metrics
 import:
 - package: github.com/m3db/m3x
-  version: a3695ff2d9696fcd9fffca878b9b544e68c4b1ef
+  version: 29bc232d9ad2e6c4a1804ccce095bb730fb1c6bc
```

Comment:
mind pulling in m3cluster too: https://github.com/m3db/m3cluster/commit/cb17a54d6902c03790286942bafa04a188d4eeb8 is the latest sha1

Reply:
Sure thing.